tests/test_uuid.py

```diff
@@ -100,9 +104,18 @@ def test_uuid7() -> None:
     uuid = uuid_utils.uuid7(1679665408)
     assert isinstance(uuid, uuid_utils.UUID)
 
+    uuid = uuid_utils.uuid7(1679665408, 999)
+    assert isinstance(uuid, uuid_utils.UUID)
+
     uuid = uuid_utils.uuid7()
     assert isinstance(uuid, uuid_utils.UUID)
 
+    ts = datetime(
+        year=2024, month=1, day=2, hour=3, minute=4, second=5, microsecond=123000
+    )
+    uuid = uuid_utils.uuid7(int(ts.timestamp()), ts.microsecond * 1_000)
+    assert uuid.timestamp == int(ts.timestamp() * 1000)
```

Comment:
Just an FYI, I changed your test to use the `timestamp` property of the `UUID` which is an extra property compared to the stdlib.

Reply:
Probably a good idea to add documentation to this project :sweat_smile: